api/v1alpha1/flinkcluster_types.go

```diff
@@ -147,6 +147,8 @@ type JobManagerSpec struct {
 	// Volume mounts in the JobManager container.
 	VolumeMounts []corev1.VolumeMount `json:"volumeMounts,omitempty"`
 
+	VolumeClaimTemplates []corev1.PersistentVolumeClaim `json:"volumeClaimTemplates,omitempty"`
```

Comment:
v1alpha1 shouldn't be updated, could you revert it? Only update v1beta1.

Reply:
@functicons fixed